### DISCUSSION TITLE
a way to compute a set of non photos jpeg images?

### DISCUSSION THREAD

**Discussion by deedjee** [alicevision/Meshroom#1868] [Ideas]
Hello,
I have a list of images of a drawing representing a character with different views in rotation
When i load this set in the program he refused the majority of the image and fail to process the rendering
Is it mandatory to use photos from a camera? it would be very interesting to work from sketch and drawing.

**Comment by FlachyJoe** [comment]
Hi,
you can try to draw your character in a virtual environment you rotate between the views for example a randomly textured cube interior as background.
I think you need to be very accurate in the angle you draw.

**Comment by natowi** [comment]
"Is it mandatory to use photos from a camera?" Yes, as you drawings are will have slight differences in form, colour and texture which will be misinterpreted by the algorithms. 

"draw your character in a virtual environment"
That is probably the best approach. You will find plenty tutorials on youtube for a software of your choice.
